sql/updates/latest-dev.sql

```diff
@@ -17,13 +20,17 @@ DROP VIEW IF EXISTS timescaledb_information.license;
 
 -- Add new function definitions, columns and tables for distributed hypertables
 DROP FUNCTION IF EXISTS create_hypertable(regclass,name,name,integer,name,name,anyelement,boolean,boolean,regproc,boolean,text,regproc,regproc);
+DROP FUNCTION IF EXISTS create_distributed_hypertable(regclass,name,name,integer,name,name,anyelement,boolean,boolean,regproc,boolean,text,regproc,regproc,integer,name[]);
```

Comment:
Is this line really needed given that `create_distribute_hypertable` didn't exist in a previous version?

Reply:
I am unsure what "previous version" means in this context. The previous beta or 1.7? I can test to remove it and see if the update script works.

Reply:
Betas are not upgradable, so the most recent version we can upgrade from is 1.7.4.

Reply:
Update CI worked without these lines, so removed them.